full pass copy edits (#17)
everywhere-computer/writing-functions-blogpost-2024: BLOGPOST.md

Running this script will produce a Wasm component with a `subtract` name prefix and a hash, for example `subtract-j54di3rspj2eewjro4.wasm`.

> Wasmify is built on top of [ComponentizeJS][componentize-js], which ingests JavaScript source code and embeds SpiderMonkey in a Wasm component to run it. Embedding SpiderMonkey and running JavaScript code comes at a size and performance cost compared to languages that can compile to WebAssembly directly; however, it enables us to empower JavaScript developers to quickly get started with writing custom functions. See [Fast(er) JavaScript on WebAssembly: Portable Baseline Interpreter and Future Plans][javascript-webassembly-post] for more information.

Reviewer: ```suggestion
Wasmify is built on top of [ComponentizeJS][componentize-js], which ingests JavaScript source code and embeds SpiderMonkey in a Wasm component to run it. Embedding SpiderMonkey and running JavaScript code comes at a size and performance cost compared to languages that can compile to WebAssembly directly, but it lets JavaScript developers get started quickly with writing custom functions. 

See [Fast(er) JavaScript on WebAssembly: Portable Baseline Interpreter and Future Plans][javascript-webassembly-post] for more information.
```

We recommend reading the [Introducing Componentize-Py][introducing-componentize-py-blog] blog post for more information on writing Python-sourced components. Also, [Introducing Componentize-Py: A Tool for Packaging Python Apps as Components][introducing-componentize-py-video] is an excellent talk that explains how `componentize-py` works.

### Workflows

We now have a set of Wasm components with arithmetic functions sourced from multiple languages. Let's run these functions together in some workflows!

> Install [EveryCLI][every-cli-npm] if you haven't already done so from the [Quickstart docs][ec-quickstart], and then we'll write a workflow.

Reviewer: ```suggestion
Install [EveryCLI][every-cli-npm], and then we'll write a workflow.
```

The first error is our WASI log reporting a "Division by zero error". The second error is an execution error from the Wasm runtime. It's a bit inscrutable, but we can see "not able to run fn divide" which tells us which function failed.

## Conclusion

In this post, we have introduced the Everywhere Computer and how you can get started writing functions and workflows for it.

> So far, to execute workflows, we've used `curl` to POST manually constructed JSON workflows to the gateway. You may have noticed from the EveryCLI start-up that it spins up a local Control Panel web interface:

Reviewer: ```suggestion
So far, to execute workflows, we've used `curl` to POST manually constructed JSON workflows to the gateway. You may have noticed from the EveryCLI starts a Control Panel web interface:
```
